cmd/finish/main.go

```diff
@@ -59,7 +59,7 @@ func main() {
 		*nexusURLFlag,
 		*nexusUsernameFlag,
 		*nexusPasswordFlag,
-		ctxt.Project, // should it be repository?
+		"ods-pipelines",
```

Comment:
Let's make this a flag and default it to this value

Reply:
Fixed